src/homePage.js

```diff
@@ -79,52 +72,49 @@ class App extends Component {
         <Button
           style={{ marginLeft: 20, marginRight: 20 }}
           variant="contained"
-          type="Submit"
+          type="submit"
         >
           Login
         </Button>
 
         <br />
 
-        <Button variant="contained">
+        <Button  type = 'button' variant="contained">
           <Link to="/registerPage">Register</Link>
         </Button>
       </FormControl>
+      </form>
     </div>
   );
 
-  render() {
-    return (
-      <div className="App">
-        <Routes>
-          <Route exact path="/registerPage" element={<RegisterForm />}></Route>
-          <Route
-            exact
-            path="/todolist"
-            element={
-              <Todolist
-                email={this.state.currentUser.email}
-                isLoggedIn={this.state.isloggedIn}
-                todolist={this.state.currentUser.taskList}
-                password={this.state.currentUser.password}
-              />
-            }
-          />
-          <Route
-            exact
-            path="/"
-            element={
-              this.state.isloggedIn ? (
-                <Link to="/todolist">Todolist</Link>
-              ) : (
-                <>{this.loginForm}</>
-              )
-            }
-          ></Route>
-        </Routes>
-      </div>
-    );
-  }
+  return (
+    <div className="App">
+      <Routes>
+        <Route exact path="/registerPage" element={<RegisterForm />}></Route>
+        <Route
+          exact
+          path="/todolist"
+          element={
+            <Todolist
+              email={currentUser.email}
+              isLoggedIn={isLoggedIn}
+            />
+          }
+        />
+        <Route
+          exact
+          path="/"
+          element={
+            isLoggedIn ? (
```

Comment:
have seperate routs for login and tasklist page, on login should redirect to a diffrent route /tasklist